prototool.yaml

```diff
@@ -7,7 +7,7 @@ protoc:
   version: 3.10.0
   includes:
     - .
-    - ../../gogo/protobuf/protobuf
+    - ../../tron-us/protobuf/protobuf
```

Comment:
Sorry this may be a late question: how do you guarantee this folder contains the right release, say v1.3.4? seems to be easy broken

Reply:
I can check if protottool provides anything, but currently, gogo version cannot be guaranteed here.

Reply:
small follow up debt then, not a big issue right now